src/main/java/mat/server/clause/QDSAttributesServiceImpl.java

```diff
@@ -29,11 +29,13 @@
 import mat.model.clause.QDSAttributes;
 import mat.server.SpringRemoteServiceServlet;
 import mat.server.util.ResourceLoader;
+import org.springframework.stereotype.Service;
 
 /**
  * The Class QDSAttributesServiceImpl.
  */
 @SuppressWarnings("serial")
+@Service

```

Comment:
It's a servlet. It should not have a Service annotation (if it's not injected somewhere else).

Reply:
Wat its not a servlet? its a spring bean.

Reply:
See what you mean now, my bad. It was a bean before but prob someone new to spring. Going to try removing it and see if it works.

Reply:
It worked no need for it to be a bean.